### PR TITLE
mark the support of foreign key as experimental

### PR DESCRIPTION
**Pull request by qiancai**
### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions](https://github.com/pingcap/docs/blob/master/CONTRIBUTING.md#guideline-for-choosing-the-affected-versions).

- [x] master (the latest development version)
- [x] v7.5 (TiDB 7.5 versions)
- [x] v7.4 (TiDB 7.4 versions)
- [x] v7.3 (TiDB 7.3 versions)
- [x] v7.1 (TiDB 7.1 versions)
- [ ] v6.5 (TiDB 6.5 versions)
- [ ] v6.1 (TiDB 6.1 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from: https://github.com/pingcap/docs/pull/15290
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [x] Might cause conflicts after applied to another branch

### CODE REVIEW

File: develop/dev-guide-sample-application-nodejs-prisma.md [hunk 1]
```diff
@@ -344,13 +344,15 @@ await prisma.player.delete({
 
 ### 外键约束与 Prisma Relation Mode
 
-对于 TiDB v6.6.0 或更高版本，推荐使用[外键约束](https://docs.pingcap.com/zh/tidb/stable/foreign-key)来替代 [Prisma Relation Mode](https://www.prisma.io/docs/concepts/components/prisma-schema/relations/relation-mode) 来实现[参照完整性](https://zh.wikipedia.org/wiki/%E5%8F%82%E7%85%A7%E5%AE%8C%E6%95%B4%E6%80%A7)检查。
+你可以使用外键约束或 Prisma Relation Mode 来检查[参照完整性](https://zh.wikipedia.org/wiki/%E5%8F%82%E7%85%A7%E5%AE%8C%E6%95%B4%E6%80%A7)：
 
-Relation Mode 是 Prisma Client 端对外键约束的模拟。该特性会对应用程序的性能产生一些影响，因为它需要额外的数据库查询来维护参照完整性。
+- [外键](/foreign-key.md)是从 TiDB v6.6.0 开始支持的实验特性，外键允许跨表交叉引用相关数据，外键约束则可以保证相关数据的一致性。
```
Comment (qiancai) [inline develop/dev-guide-sample-application-nodejs-prisma.md:349]:
```suggestion
- [外键](/foreign-key.md)是 TiDB 从 v6.6.0 开始支持的实验特性，外键允许跨表交叉引用相关数据，外键约束则可以保证相关数据的一致性。
```

Comment (qiancai) [inline develop/dev-guide-sample-application-nodejs-prisma.md:349]:
```suggestion
- [外键](/foreign-key.md)是 TiDB 从 v6.6.0 开始支持的实验特性，外键允许跨表交叉引用相关数据，外键约束则可以保证相关数据的一致性。
```

File: releases/release-6.6.0.md [hunk 2]
```diff
@@ -172,7 +172,7 @@ TiDB 版本：6.6.0-[DMR](/releases/versioning.md#开发里程碑版本)
 
 ### SQL 功能
 
-* 支持兼容 MySQL 语法的外键约束 [#18209](https://github.com/pingcap/tidb/issues/18209) @[crazycs520](https://github.com/crazycs520)
+* 支持兼容 MySQL 语法的外键约束（实验特性）[#18209](https://github.com/pingcap/tidb/issues/18209) @[crazycs520](https://github.com/crazycs520)
```
Comment (Oreoxmt) [inline releases/release-6.6.0.md:175]: L334 也需要加一下实验特性

```
* 支持兼容 MySQL 语法的外键约束 [#18209](https://github.com/pingcap/tidb/issues/18209) @[crazycs520](https://github.com/crazycs520)
```

File: develop/dev-guide-sample-application-nodejs-prisma.md
Comment (Oreoxmt) [inline develop/dev-guide-sample-application-nodejs-prisma.md:1]: 这篇也需要加上实验特性说明
https://github.com/pingcap/docs-cn/blob/3b1d4ad4faa6158e73fd8f521136c07a979801b2/develop/dev-guide-sample-application-nodejs-typeorm.md?plain=1#L338-L340

File: foreign-key.md [hunk 3]
```diff
@@ -7,9 +7,10 @@ summary: TiDB 数据库中外键约束的使用概况。
 
 从 v6.6.0 开始，TiDB 支持外键以及外键约束功能，外键允许跨表交叉引用相关数据，外键约束则可以保证相关数据的一致性。
 
-> **注意：**
+> **警告：**
 >
-> 外键功能通常适用于为中小规模的数据提供完整性和一致性约束校验，但是在大数据量和分布式数据库系统下，使用外键可能会导致严重的性能问题，并对系统产生不可预知的影响。如果计划使用外键，请进行充分验证后谨慎使用。
+> - 外键功能目前为实验特性，不建议在生产环境中使用。
```
Comment (Oreoxmt) [inline foreign-key.md:12]:
```suggestion
> - 外键功能目前为实验特性，不建议在生产环境中使用。该功能可能会在未事先通知的情况下发生变化或删除。如果发现 bug，请在 GitHub 上提 [issue](https://github.com/pingcap/tidb/issues) 反馈。
```

File: develop/dev-guide-sample-application-nodejs-prisma.md
Comment (qiancai) [reply develop/dev-guide-sample-application-nodejs-prisma.md:1]: Done in https://github.com/pingcap/docs-cn/pull/15391/commits/4752b0d555db677856d3f6521f10646ef6f67c77

File: releases/release-6.6.0.md (hunk 2 above)
Comment (qiancai) [reply releases/release-6.6.0.md:175]: Done in https://github.com/pingcap/docs-cn/pull/15391/commits/4752b0d555db677856d3f6521f10646ef6f67c77

**Review by Oreoxmt — APPROVED** [review]

**Pull request merged**